test/services/users.test.js

```diff
@@ -23,7 +23,7 @@ describe('User service tests', function () {
         .insert(user)
         .then(function (newUser) {
           const expectedUserProps = ['firstname', 'lastname', 'email', 'isActive', 'username', 'id', 'createdAt', 'updatedAt'];
-          Object.keys(newUser).should.eql(expectedUserProps);
+          should(Object.keys(newUser)).containDeep(expectedUserProps);
```

Comment:
this is kind of `expect(foo).tobe(bar)` syntax
why do you like it? and why it is called should in this case ? 

I expect should style like
`foo.should.be(bar)`

Reply:
I am not that opinionated on the style — both are ok. 

However I prefer `should(object).have.properties(…)` over `object.should.have.properties` is because the former is more explicit than the latter.

`should()` is an explicit function reference that has been imported on the top of the file.
`const should = require('should');`

On the other hand, the `should` property on tested objects is implicitly injected into the object. This feels like magic.

Reply:
ok, I have no preference, just asking